cmd/tiller-proxy/internal/handler/handler_test.go

```diff
@@ -49,7 +50,7 @@ func TestErrorCodeWithDefault(t *testing.T) {
 		{fmt.Errorf("This is an unexpected error"), http.StatusUnprocessableEntity, http.StatusUnprocessableEntity},
 	}
 	for _, s := range tests {
-		code := errorCodeWithDefault(s.err, s.defaultCode)
+		code := handlerutil.ErrorCodeWithDefault(s.err, s.defaultCode)
```

Comment:
since this test is just testing `ErrorCodeWithDefault`, it's better to move this test to kg/handlerutil/handlerutil_test.go

Reply:
Yes, I agree.